include/umap.h

```diff
@@ -34,15 +34,14 @@ void* umap( void*  addr,    /* See mmap(2) */
             int    fd,      /* See mmap(2) */
             off_t  offset   /* See mmap(2) */
         );
+int uunmap( void*  addr,    /* See mmap(2) */
```

Comment:
Not sure why uunmap() moved (it was the only change in this file), but this leads me to ask why we don't have a uunmap_mf()?